doc/index.rst

```diff
@@ -1,8 +1,42 @@
-Plutus
-======
+Plutus Core and Plutus Tx user guide
+==================================================
 
-Plutus provides the support for writing scripts that run on the Cardano blockchain.
-To read more about the overall architecture of writing applications using Plutus, see :ref:`what_is_the_plutus_platform`.
+Plutus Core
+---------------------
+
+The Plutus project consists of Plutus Core, the programming language used for 
+scripts on Cardano; tooling and compilers for compiling various intermediate 
+languages into Plutus Core; and Plutus Tx, the compiler that compiles the Haskell 
+source code into Plutus Core to form the on-chain part of a contract application. 
+All of this is used in combination to write Plutus Core scripts that run on the 
+Cardano blockchain. The Cardano ledger is able to interact with compiled Plutus Core 
+scripts through the ledger interface. 
+
+This documentation introduces the Plutus Core programming language and programming 
+with Plutus Tx. It includes explanations, tutorials, how-to instructions, 
+troubleshooting, and reference information. 
+
+The intended audience of this documentation includes people who want to implement 
+smart contracts on the Cardano blockchain. This involves using Plutus Tx to write 
+scripts, requiring some knowledge of the Haskell programming language. 
+
+.. note::
+   See the `public Plutus libraries documentation <https://playground.plutus.iohkdev.io/doc/haddock/>`_ 
+   to access Haddock-generated documentation of all the code. 
+
+This guide is also meant for certification companies, certification auditors, 
+and people who need to reference specifications. See, for example: 
```

Comment:
```suggestion
and people who need an accurate specification. See, for example: 
```
I think this sounds a bit better.

Reply:
Thanks for your suggestion. I'll incorporate that.